src/qt/forms/receiverequestdialog.ui

```diff
@@ -6,68 +6,233 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>487</width>
-    <height>597</height>
+    <width>413</width>
+    <height>229</height>
    </rect>
   </property>
-  <layout class="QVBoxLayout" name="verticalLayout_3">
-   <item>
-    <widget class="QRImageWidget" name="lblQRCode">
-     <property name="sizePolicy">
-      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
-       <horstretch>0</horstretch>
-       <verstretch>0</verstretch>
-      </sizepolicy>
-     </property>
-     <property name="minimumSize">
-      <size>
-       <width>300</width>
-       <height>320</height>
-      </size>
-     </property>
-     <property name="toolTip">
-      <string>QR Code</string>
+  <property name="windowTitle">
+   <string>Request payment to ...</string>
+  </property>
+  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
+   <property name="sizeConstraint">
+    <enum>QLayout::SetFixedSize</enum>
+   </property>
+   <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
+    <widget class="QRImageWidget" name="qr_code">
+     <property name="text">
+      <string notr="true">QR image</string>
+     </property>
+     <property name="textInteractionFlags">
+      <set>Qt::NoTextInteraction</set>
+     </property>
+    </widget>
+   </item>
+   <item row="1" column="0" colspan="2">
+    <widget class="QLabel" name="payment_header">
+     <property name="font">
+      <font>
+       <weight>75</weight>
+       <bold>true</bold>
+      </font>
+     </property>
+     <property name="text">
+      <string>Payment information</string>
+     </property>
+     <property name="textInteractionFlags">
+      <set>Qt::NoTextInteraction</set>
+     </property>
+    </widget>
+   </item>
+   <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
+    <widget class="QLabel" name="uri_tag">
+     <property name="font">
+      <font>
+       <weight>75</weight>
+       <bold>true</bold>
+      </font>
+     </property>
+     <property name="text">
+      <string notr="true">URI:</string>
```

Comment:
This SHOULD be translated...

Reply:
"URI" is a ubiquitous international abbreviation which does not require any translation, IMO.
Refs:
- https://www.wikipedia.org/ (in different languages)
- [`src/qt/locale/*.ts`](https://github.com/bitcoin/bitcoin/tree/master/src/qt/locale)

Reply:
What about languages which don't use Latin characters?

eg https://ur.wikipedia.org/wiki/%DB%8C%D9%88_%D8%A2%D8%B1_%D8%A2%D8%A6%DB%8C

Reply:
> What about languages which don't use Latin characters?

[qt/locale/bitcoin_ur.ts](https://github.com/bitcoin/bitcoin/blob/master/src/qt/locale/bitcoin_ur.ts) does not translate "URI".

My native language does not use Latin characters, and "URI", "HTTP" are in use.

Reply:
Probably best to let translators decide that. I doubt 99.9% of native English speakers even know what URI means.